### DISCUSSION TITLE
Theater of Blood "Blacklist" Suggestion

### DISCUSSION THREAD

**Discussion by hard-bush** [runelite/runelite#16286] [Q&A] [answered]
Ignored players RSN's show red (or whatever color) on the "orb order party list"- while inside the TOB, as well as on the team creation board. 

This will allow players to be aware and decide whether to raid with the countless tobbers that have a record of trolling and known sabotage that have been added to their ignore list to track name changes and just the sheer number of them. please discuss and implement if worthy 🫡🤞

**Comment by geheur** [comment, marked as answer]
The ToB Notice Board plugin from the plugin hub does some of these things..

**Reply by hard-bush** [reply]
Il check it out. thanks for info ❤️

**Reply by hard-bush** [reply]
checked it out and its awesome, would be a nice touch to add red names to orb order in case u dont watch board for entire recruitment process/or decide to give red names a second chance, and are now able to keep a better eye on them while you tryhard

**Comment by hard-bush** [comment]
The plugin doesnt seem to work on hard mode!!?

**Reply by Nightfirecat** [reply]
Please take this discussion/feature requests to that plugin's repository; the RuneLite team doesn't offer support for hub plugins.

**Reply by hard-bush** [reply]
will do shorty. thanks for guidance.
